TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -1390,43 +1389,26 @@ currentIndex < (lastIndex+1) && (line[currentIndex + 2] == '"' || line[currentIn
 
                 // PROBLEMS : 
                 // 123 is a valid user defined word (for section & paragraph names) AND a valid numeric literal.
-                // 123E-4 is a valid user defined word (for any type of name) AND a valid numeric literal (floating point format).
-                // 123-456 is a valid user defined word (fol section & paragraph names), AND it could be interpreted as two numeric literals,
-                //   NB: it is NOT a valid subtraction (minus must be preceded and followedf by space) but we would like to display a nice error 
-                //   message informing the user that these spaces are mandatory, because a subtraction was most likey intended in this case.
-                // 000010-000050 should be interpreted as a range of numbers (indicated by separating the two bounding numbers of the range 
-                //   by a hyphen) in sequence-number-fields of compiler directive statements (ex: DELETE).
-
+                // 000010-000050 is interpreted as a UserDefinedWord
+                //Depending on the context it could be
+                // - a range of numbers (for DELETE_CB)
+                //     - See sequence-number-fields of compiler directive statements
+                // - a paragraph name
+                //The scanner will then always match 000010-000050 as a UserDefinedWord.
+                //The check to be sure that a variable name contains at least alphabetic char
+                //must be done at CodeElement level.
```

Comment:
This is done in this PR right ? So the comment should be more affirmative: 'must be done' becomes 'is done'

Reply:
Done in e705b29f87cea7244fae092d8bc7488e8de90876